docs/non-normative/k8s-migration.md

```diff
@@ -422,6 +425,60 @@ The changes in these metrics are the following:
 
 <!-- prettier-ignore-end -->
 
+### K8s Pod Memory metrics
+
+The K8s Pod memory metrics implemented by the Collector and specifically the
+[k8scluster](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.119.0/receiver/k8sclusterreceiver/documentation.md)
+receiver were introduced as semantic conventions in
+[#1490](https://github.com/open-telemetry/semantic-conventions/issues/1490).
+
+The changes in these metrics are the following:
+
+<!-- prettier-ignore-start -->
+
+| Old (Collector) ![changed](https://img.shields.io/badge/changed-orange?style=flat) | New                                            |
+|------------------------------------------------------------------------------------|------------------------------------------------|
+| `k8s.pod.memory.page_faults`                                                         | `k8s.pod.memory.paging.faults` with attribute `system.paging.type` |
+| `k8s.pod.memory.major_page_faults`                                                   | `k8s.pod.memory.paging.faults` with attribute `system.paging.type` |
```

Comment:
Should we mention the attribute values? Otherwise, it's unclear what's the replacement.

Also the PR description still uses the old naming. Can you please update it?